buildconfig/Jenkins/buildscript

```diff
@@ -38,6 +38,12 @@ else
   BUILD_CONFIG="Release"
 fi
 
+# BUILD_PACKAGE can be provided as a job parameter on the pull requests
+# except RHEL7 as we always need to it to do the system tests
+if [[ ${JOB_NAME} == *pull_requests* ]]; then
+  BUILDPKG=${BUILD_PACKAGE}
```

Comment:
This will overwrite the `BUILDPKG` variable with an empty string if the `BUILD_PACKAGE` environment variable isn't set. Add `if [[ ! -z "$BUILD_PACKAGE" ]]; then` inside the `if [[ ${JOB_NAME} ...`.